lms/djangoapps/courseware/tests/test_lti_integration.py

```diff
@@ -187,24 +187,24 @@ def test_lti_rest_bad_course(self):
         for bad_course_id in bad_ids:
             lti_rest_endpoints_url = 'courses/{}/lti_rest_endpoints/'.format(bad_course_id)
             response = self.client.get(lti_rest_endpoints_url)
-            self.assertEqual(404, response.status_code)
+            assert 404 == response.status_code
 
     def test_lti_rest_listing(self):
         """tests that the draft lti module is part of the endpoint response"""
         request = mock.Mock()
         request.method = 'GET'
         response = get_course_lti_endpoints(request, course_id=text_type(self.course.id))
 
-        self.assertEqual(200, response.status_code)
-        self.assertEqual('application/json', response['Content-Type'])
+        assert 200 == response.status_code
+        assert 'application/json' == response['Content-Type']
```

Comment:
is `pyupgrade` also taking care of replacing `unittest` assertions with `pytest` ones, or is this the result of merge conflicts?

Reply:
I changed these manually.